packages/core/src/modules/connections/services/ConnectionService.ts

```diff
@@ -177,6 +178,7 @@ export class ConnectionService {
       imageUrl: config?.myImageUrl ?? this.config.connectionImageUrl,
     })
 
+    connectionRecord.autoAcceptConnection = config?.autoAcceptConnection
```

Comment:
I think this could overwrite the previously set autoAcceptConnection state.

If receiveInvitation had autoAccept set to false and you don't provide it for acceptInvitation, it will be set to undefined.

If the global config is set to true, having false and undefined can have different meaning.

The problem then arises how do you clear autoAccept (null value?), but I think that's another issue to solve

Reply:
I see. What about adding the following condition?

```ts
if (config?.autoAcceptConnection !== undefined || config?.autoAcceptConnection !== null) {
  connectionRecord.autoAcceptConnection = config?.autoAcceptConnection
}
```

That wouldn't still solve clearing `autoAccept` but I could be good enough for now.

Reply:
Yeah that looks good to me. I think just `!== undefined` should be fine, as `null` is not an allowed value

Reply:
Better safe than sorry 😄  Actually, I think `null` can appear there. What if the user of the framework doesn't use TS?